spec/requests/api/querying_spec.rb

```diff
@@ -599,7 +598,7 @@ def create_vms_by_name(names)
       run_get vms_url, :expand => "resources,software"
 
       expect_query_result(:vms, 1, 1)
-      expect_result_resources_to_include_keys("resources", %w(id href guid name vendor software))
```

Comment:
I'm not sure I follow why these keys disappeared as a result of this change and whether that was intended?